inst/Description.qml

```diff
@@ -28,9 +28,9 @@ Description
 	}
 	Analysis
 	{
-		title:	qsTr("Intraclass Correlation")
-		qml: 	"IntraclassCorrelation.qml"
-		func: 	"IntraclassCorrelation"
+		title:	qsTr("Rater Agreement")
```

Comment:
I am uncertain what I think about the term "Rater Agreement", since some of the ICCs might also be used for test-retest reliability. I suppose "Rater Agreement" would also include coefficients, such as Cohens's kappa or Fleiss' Kappa. Maybe I would leave it at "Intraclass Correlation Coefficients" for this analysis, and add an extra analysis with something like "Agreement Coefficients", which would then include everything not covered by the ICCs. Maybe @ej has an opinion about this. If you decide to change this, it will have consequences for the module menu as well.

Reply:
Thanks for the feedback @juliuspf, yeah the reason I went with Rater Agreement was to be able to include Cohen's Kappa (and the likes) at some point in the future, but maybe it does make more sense to split these into separate analyses... either works for me 🤷 👍️

Reply:
I suppose whenever the data we would use for an analysis is fundamentally different, it makes sense to do a new analysis. And I guess for kappa the data is neither ordinal nor metrical. But this is just my opinion.